pkg/cmd/generate/permissions.go

```diff
@@ -171,7 +171,7 @@ func ensureServiceAccount(saNamespace string) newSubjectFunc {
 
 // ensureUserIdentityAndGroups ensures that all - User, Identity, IdentityMapping, and Group manifests - exist
 func ensureUserIdentityAndGroups(IDs []string, groups []string) newSubjectFunc {
-	return func(ctx *clusterContext, cache objectsCache, subjectBaseName, targetNamespace string, labels map[string]string) (rbacv1.Subject, error) {
+	return func(ctx *clusterContext, cache objectsCache, subjectBaseName, _ string, labels map[string]string) (rbacv1.Subject, error) {
```

Comment:
```suggestion
	return func(ctx *clusterContext, cache objectsCache, subjectBaseName, labels map[string]string) (rbacv1.Subject, error) {
```

can't we remove that parameter itself?

Reply:
no, because it's used in a generic way and there is another version of this function which ensures serviceAccounts that needs the target namespace

Reply:
thanks for clarifying

Reply:
alternatively we could set the namespace on the user and identity or maybe those are cluster scoped resources

Reply:
this can be still overridden for a specific SA